core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -433,7 +440,8 @@ private List<MediaBrowserCompat.MediaItem> loadChildrenSynchronous(@NonNull Stri
             long feedId = Long.parseLong(parentId.split(":")[1]);
             Feed feed = DBReader.getFeed(feedId);
             feedItems = DBReader.getFeedItemList(feed, FeedItemFilter.unfiltered(), feed.getSortOrder());
-        } else if (parentId.equals(getString(R.string.recently_played_episodes))) {
+        } else if (parentId.equals(getString(R.string.recently_played_episodes))
+                || parentId.equals(getString(R.string.current_playing_episode))) {
```

Comment:
If this does the same as an existing screen, why do we need to have it twice?

Reply:
This is a workaround for not receiving `BrowserRoot.EXTRA_RECENT` on this line 

```
        if (rootHints != null && rootHints.getBoolean(BrowserRoot.EXTRA_RECENT)) {
```

Reply:
Then how about changing the `EXTRA_RECENT` part to use the same `parentId` instead of having two different IDs do the same thing?

Reply:
done.  I think this is a good compromise for folks in Android Auto